website/public/sitemap.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@enisdenjo should I add this file to gitignore?

Reply:
Yeah, good catch! The sitemap is generated on each deploy and is therefore not necessary to commit.